### PR TITLE
hotspot & name convention update

### PR DESCRIPTION
**Pull request by DannyS03**
https://github.com/OpenLiberty/guides-common/issues/352
https://github.com/OpenLiberty/guides-common/issues/308

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -138,21 +138,21 @@ include::finish/src/main/liberty/config/server.xml[tags=**]
...
+By pairing this annotation with the [hotspot=ApplicationScoped]`ApplicationScoped` context from the Contexts and 
 Dependency Injections API, the bean is discovered automatically when the http://localhost:9080/health[http://localhost:9080/health^] 
 endpoint receives a request.
 
-The [hotspot=12-20]`call()` method is used to return the health status of a particular service. 
-In this case, you are simply checking if the server name is [hotspot=13]`defaultServer` and 
-returning [hotspot=18-19]`UP` if it is, and [hotspot=14-16]`DOWN` otherwise. The 
-[hotspot=14]`HealthCheckResponse.named()` method is used to indicate what service the health 
-check is done for. Overall, this is a very simple implementation of the [hotspot=12-20]`call()` 
+The [hotspot=HealthCheckResponse]`call()` method is used to return the health status of a particular service.
```
Comment (gkwan-ibm) [inline README.adoc]: The `call()` hotspot HealthCheckResponse does not highlight as the master site. The master highlight the whole `call()` method.

Comment (DannyS03) [reply README.adoc]: Updated.

Comment (gkwan-ibm) [reply README.adoc]: The second `call()` hotspot does not work.

Comment (DannyS03) [reply README.adoc]: Fixed.

**Comment by DannyS03** [comment]
- verified on `qa`.
- LGTM.

**Pull request merged**